packages/docusaurus-theme-classic/src/theme/BlogPostItem/index.tsx

```diff
@@ -127,9 +127,20 @@ function BlogPostItem(props: Props): JSX.Element {
     );
   };
 
+  const metaTags = isBlogPostPage
+    ? {
+        'og:type': 'article',
+        'article:published_time': date,
+        ...(authorURL ? {'article:author': authorURL} : {}),
+        ...(frontMatter.tags
+          ? {'article:tag': frontMatter.tags.join(',')}
```

Comment:
Shouldn't each tag be in separate meta tag? https://stackoverflow.com/questions/36230298/how-to-add-multiple-tags-article-tag-in-facebook-open-graph

Reply:
I looked at usage it seems to be more commonly done this way.  Take a look at how the Guardian are doing this:

https://www.theguardian.com/money/2021/aug/18/england-and-wales-on-track-for-biggest-train-fare-rise-since-2012

```html
<meta property="article:tag" content="Rail fares,Rail transport,UK news,Transport policy,Business,Transport,Politics">
```

Reply:
If it works that way, alright then.